compiler/cpp/src/generate/t_py_generator.cc

```diff
@@ -384,6 +406,55 @@ void t_py_generator::init_generator() {
     py_autogen_comment() << endl <<
     py_imports() << endl <<
     "from .ttypes import *" << endl;
+
+  construct_object_dependent_graph();
+}
+
+/**
+ * Construct object dependent graph to determine object generate order
+ * if cycle dependent is detected, generate process will be aborted
+ * object dependent example
```

Comment:
But your test case is perfectly valid Thrift IDL:
```
struct A {
  1: B b;
}

struct B {
  1: A a;
}
```

One can do this with Thrift. Is there no way to support that in Python? Something like forward-declaration of types maybe?